paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -41,16 +41,14 @@ fun detectEnvironment(): Environment {
       .last()
 
   val configLines = File("build/intermediates/paparazzi/resources.txt").readLines()
-  val packageName = configLines[0]
-  val resDir = configLines[1]
+  val (packageName, resDir) = configLines
 
   return Environment(platformDir, userDir, resDir, packageName)
 }
 
 private fun checkInstalledJvm() {
-  val jvmVendor = System.getProperty("java.vendor")
-  val jvmVersion = System.getProperty("java.version")
-  if (jvmVendor == null || jvmVersion == null) return // we tried...
+  val jvmVendor = System.getProperty("java.vendor") ?: return
+  val jvmVersion = System.getProperty("java.version") ?: return
```

Comment:
Unrelated to this PR but should we throw exception here? If there's no `java.vendor` or `java.version` are we still guaranteed that JVM is available?